pkg/machine/config.go

```diff
@@ -19,6 +20,65 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type GuestOS uint32
+
+const (
+	CoreOS GuestOS = iota
+	// Any unknown Linux
+	GenericLinux
```

Comment:
It would be better for everyone if this was just called "Linux", like in `uname` and like everywhere else (like Lima)

Reply:
I'm not attached to any of the naming, but I wanted to decouple specific-Linux-distribution from unknown-may-not-work-but-let's-try-it-anyway-Linux-distribution.  Happy to call this Linux (though, note, it's a distribution here, not a kernel, so `uname` output is not relevant, `lsb_release` output might be) if that's preferable.  It's there so that people bringing their own Linux distro can fall through into generic Linux code paths where appropriate.